### ISSUE TITLE
Cnab400 Bradesco

### ISSUE THREAD

**Issue by michael-nel** [BoletoNet/BoletoNetCore#58]
Fiz uns testes na geracao do arquivo CNAB400 Bradesco e pelo oque vi não foi implementado o cancelamento. Correto?

**Comment by olavorn** [comment]
Todas as implementações são feitas via partial classes nas pastas. 

A implementação CNAB400 do bradesco está em https://github.com/BoletoNet/BoletoNetCore/blob/master/BoletoNetCore/Banco/Bradesco/BancoBradesco.CNAB400.cs

**Comment by michael-nel** [comment]
Sim.. o trecho que trata o cancelamento do CNAB esta em branco. Vou realizar as implementações necessarioas.

                reg.Adicionar(TTiposDadoEDI.ediAlphaAliEsquerda_____, 0094, 001, 0, Empty, ' ');
                reg.Adicionar(TTiposDadoEDI.ediAlphaAliEsquerda_____, 0095, 010, 0, Empty, ' ');
                reg.Adicionar(TTiposDadoEDI.ediAlphaAliEsquerda_____, 0105, 001, 0, Empty, ' ');
                reg.Adicionar(TTiposDadoEDI.ediAlphaAliEsquerda_____, 0106, 001, 0, Empty, ' ');
                reg.Adicionar(TTiposDadoEDI.ediAlphaAliEsquerda_____, 0107, 002, 0, Empty, ' ');

**Comment by olavorn** [comment]
Blz, só fazer um PR que o carlos aprova.


Olavo Rocha Neto
_________________________________
Exodus
Product VP - Exodus Finanças
www.exodusfinancas.com.br
www.facebook.com/ExodusFinancas
blog.exodusfinancas.com.br

gTalk: olavo@exodus.eti.br
skype: neto_fire
Fone: (91) 98832-9663


Em dom., 13 de set. de 2020 às 10:48, Michael Nel <notifications@github.com>
escreveu:

**Comment by michael-nel** [comment]
Realizei a implementação necessaria, porem não tenho permissão para criar uma branch e realizar o pullrequest.

**Comment by carloscds** [comment]
@michael-nel Você precisa fazer um fork para sua conta do GitHub, fazer as alterações lá e depois enviar um Pulll Request.
